ra_aid/agents/ciayn_agent.py

```diff
@@ -102,7 +102,7 @@ def _build_prompt(self, last_result: Optional[str] = None) -> str:
 You must ONLY use ONE of the following functions (these are the ONLY functions that exist):
 
 <available functions>
-{"\n\n".join(self.available_functions)}
+{" ".join(self.available_functions)}
```

Comment:
Not sure if it matters, but this could be extracted to a variable instead of replacing the spaces with newlines.